src/Components/Quizz/Quizz.css

```diff
@@ -0,0 +1,63 @@
+
+.contentQuizz {
+    display: flex;
+    flex-direction: column;
+    padding: 10px;
+    background-image: url("http://blogs.discovermagazine.com/outthere/files/2019/07/cme_lg-1024x674.jpg");
+    background-size: cover ;
+  }
+
+
+.text-focus-in {
+	-webkit-animation: text-focus-in 1s cubic-bezier(0.550, 0.085, 0.680, 0.530) both;
```

Comment:
You don't need to write -webkit-. `create-react-app` uses `autoprefixer` which adds vendor prefixes atuomatically.